lightkurve/targetpixelfile.py

```diff
@@ -831,6 +833,84 @@ def get_bkg_lightcurve(self, aperture_mask=None):
                           flux=np.nansum(self.flux_bkg[:, aperture_mask], axis=1),
                           flux_err=self.flux_bkg_err)
 
+
+    def _get_hdu(image, extension):
+        """Checks if an image is a single FITS image, or a FITS file,
+        and returns the appropriate PrimaryHDU.
+
```

Comment:
Please explain in the docstring when someone may actually want to use this function?  It's not 100% clear to me that we need this to be a function, as opposed to being a `if-else` statement when you retrieve the WCS further below?

We want to make sure all functions inside `KeplerTargetPixelFile` have a very clear purpose, otherwise users may be confused and our code will be harder to maintain.

Reply:
Hi Geert, thank you for your feedback! I agree that this function could be misleading/it is unclear what the function's purpose is, so I will move the if-else statements into the from_fits and _get_pos_corrs functions, to minimise confusion.